internal/builder/external_fields.go

```diff
@@ -37,10 +37,20 @@ func resolveExternalFields(packageRoot, destinationDir string) error {
 		return errors.Wrap(err, "can't create field dependency manager")
 	}
 
-	fieldsFile, err := filepath.Glob(filepath.Join(destinationDir, "data_stream", "*", "fields", "*"))
+	fieldsPaths, err := filepath.Glob(filepath.Join(destinationDir, "data_stream", "*", "fields"))
```

Comment:
Umm, wdyt about leveraging Glob directly here?
```suggestion
	fieldsPaths, err := filepath.Glob(filepath.Join(destinationDir, "data_stream", "*", "fields", "*.yml"))
```

Reply:
That makes sense. I originally expected having to filter directories for multiple extensions, but turned out that's not the case most of the time. The helper function is unnecessary.